app/json/slate_configs.json

```diff
@@ -28,6 +28,35 @@
       }
     ]
   },
+  {
+    "id": "2389f563-bd42-411d-bca2-0966638053e8",
+    "displayName": "Editors' Picks",
+    "description": "Curated items that are over a week old without syndicated",
```

Comment:
Might be good to add a note about how this editors' picks differs from the other three